### PR TITLE
chore: upgrade searchbox / add suggestions / fix theme

### PR DESCRIPTION
**Pull request by rjborba**
This PR:
- Upgrades Orama components version. It fixes several styles issue and others.
- Uses dynamic theme so SearchBox colors matches the selected theme.
- Adds chat messages suggestions

image

image

### CODE REVIEW

File: app/components/DocsLayout.tsx [hunk 1]
```diff
@@ -321,6 +322,10 @@ export function DocsLayout({
   const versionConfig = useVersionConfig({ versions })
   const menuConfig = useMenuConfig({ config, frameworks, repo })
 
+  const { mode: themeMode } = useThemeStore()
+
+  const oramaThemeMode = themeMode === 'auto' ? 'dark' : themeMode
+
```
Comment (tannerlinsley) [inline app/components/DocsLayout.tsx:328]: You could detect auto here with a media match as done in other areas

**Review by tannerlinsley — CHANGES_REQUESTED** [review]

File: app/components/DocsLayout.tsx (hunk 1 above)
Comment (rjborba) [reply app/components/DocsLayout.tsx:328]: Hi @tannerlinsley nice catch.
Actually Orama components does it internally when "system" is set to colorScheme.
I've updated the code so "auto" translates to "system". Is it good enough?

Comment (rjborba) [reply app/components/DocsLayout.tsx:328]: @tannerlinsley, hey! Just checking if this is enough for you request. Thanks!

**Comment by tannerlinsley** [comment]
Yep! Just resolve conflicts and we’re good

**Comment by rjborba** [comment]
> Yep! Just resolve conflicts and we’re good

@tannerlinsley We should good to go now, then. Thanks! :)

**Comment by rjborba** [comment]
@tannerlinsley Hi! Conflicts are fixed. Can we move on with this?

**Pull request merged**